### DISCUSSION TITLE
v4.0 release?

### DISCUSSION THREAD

**Discussion by MoshiMoshi0** [d2phap/ImageGlass#184] [General] [answered]
Any ETA on v4.0 release? It was supposed to be released on 15th.
Had to replace F5 key twice already.

**Comment by d2phap** [comment]
Hi @devcompl

Thanks for your question. 
Because there is a minor issue which is just discovered in the last minutes, the release date is postponed to 23rd April. 
Sorry for your waiting time :)

**Comment by d2phap** [comment, marked as answer]
ImageGlass 4 is released today! 👍
